docs/pages/cli/verify.mdx

```diff
@@ -0,0 +1,61 @@
+# mud verify
+
+This command uploads the source code for the deployed contracts to a public repository, such as a block explorer.
+The repository then compiles the source code and verifies that the compiled version is identical to what is deployed on-chain.
+
+## Using the command
+
+1. Before you run `mud verify` you need to create a [Foundry profile](https://book.getfoundry.sh/reference/config/overview).
```

Comment:
you don't _need_ to, it just makes it a bit easier to pass in `--profile` to set various foundry options rather than individual things like `--rpc`